release-team/role-handbooks/enhancements/README.md

```diff
@@ -75,43 +78,123 @@ It is important that this process be followed and documentation remain up-to-dat
 | 0 | Pre-Collection | Ping any owners on `lifecycle/frozen` label from all issues to see if there is any planned work or if the issue is no longer relevant to Kubernetes. |
 | 0 | Pre-Collection | Send an email to the [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) mailing list with a call for enhancements. View prior examples for references ([1](https://groups.google.com/forum/#!msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ) [2](https://groups.google.com/forum/#!msg/kubernetes-dev/T-kIHtgS5J4/ZjKTsCf_AwAJ;context-place=msg/kubernetes-dev/vGSBzUUWp9g/bFa1re3ZAgAJ)). |
-| 1 | Collection | Find Issues that have been marked with the current milestone and apply the `tracked/yes` label and add it to the tracking sheet. Ensure they provide the required information and labels to be accepted into the milestone. |
-| 1 | Collection | Go to every open issue that is marked with `tracked/no` and ping the owner to see if the enhancement will be included or graduating stages in the current milestone. If there is no graduation, leave as-is. If the enhancement is graduating, remove `tracked/no`, add `tracked/yes`, add to the current milestone, and add to the tracking sheet. Another great task for shadow help or some automation. |
+| 1 | Collection | Go to every open issue that is marked with `tracked/no` and ping the owner to see if the enhancement should be included in the current milestone. If there is no graduation or major change, leave as-is. If the enhancement is graduating or undergoing a major change, remove the `tracked/no` label and add `tracked/yes`, add the issue to the current milestone, and set it's status to `Tracked` within the tracking sheet if it meets the requirements. If it is missing some  of the requirements (e.g. KEP in a provisional state), let the issue owner know and add it as `At Risk`. Another great task for shadow help or some automation. |
 | 1 | Collection | Verify issues have k/k PRs associated so they can be referenced and easily tracked. This is going to be critical come Enhancement Freeze and Code Freeze to see the status of the code. |
 | 1 | Collection | Work with the Release Lead to attend every SIG meeting over the span of 2-3 weeks. Take 2 minutes to introduce yourself, talk about release information, and relay information about issues that haven't been removed from the previous milestone or call out issues that may need some attention. |
-| 1 | Collection | New issues will be created. Take the opportunity to apply correct labels, milestone information, and add to the sheet if necessary. |
+| 1 | Collection | New issues will be created. Take the opportunity to apply correct labels, milestone information, and add it to the `data` tab within the sheet. If it's not being tracked for the current release, adding the information will help a later release team. |
```

Comment:
Not sure if this means "new issues will be created over a course of the release cycle"? 

If so, I'd change the wording to:

`New issues may be added during the release cycle. Take the opportunity to apply the correct labels, milestone information, and add them to the data tab within the sheet...`

Reply:
changed 👍